### PR TITLE
json feed add indicator_type to rawjson + test improvement

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
* Add an option that the `_indicator_type` is added to the rawjson so it is easy to use mapping of the auto detect value.
* Check in jsonfeed **Test** that *Indicator Type* and *Auto detect indicator type* are NOT both set.
* Fix an issue with `limit`

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedJSON/Integrations/FeedJSON/FeedJSON.py [hunk 1]
```diff
@@ -12,9 +12,14 @@ def main():
...
             'extractor': params.get('extractor'),
             'indicator': params.get('indicator', 'indicator'),
+            'rawjson_include_indicator_type': params.get('rawjson_include_indicator_type'),
         }
     }
 
+    if params.get('auto_detect_type') and params.get('indicator_type'):
+        return_error('Indicator Type should not be set if Auto Detect Indicator Type is checked.'
+                     ' Either use Auto Detect or set manually the Indicator Type.')
+
```
Comment (teizenman) [inline Packs/FeedJSON/Integrations/FeedJSON/FeedJSON.py:22]: We can make it more general by adding it to: https://github.com/demisto/content/blob/master/Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py#L252

WDYT?

Comment (glicht) [reply Packs/FeedJSON/Integrations/FeedJSON/FeedJSON.py:22]: I didn't want to introduce a breaking change across all feeds that use this module. I would rather leave it in json feed. Other feeds usually set this anyway in a hard coded way.

**Review by teizenman — APPROVED** [review]

File: Packs/FeedJSON/Integrations/FeedJSON/FeedJSON.py (hunk 1 above)
Comment (glicht) [reply Packs/FeedJSON/Integrations/FeedJSON/FeedJSON.py:22]: anyway also doing a commit to only do this validation on test-module command.

**Comment by glicht** [comment]
@amshamah419 can you take a look at the JSON Feed release notes: https://github.com/demisto/content/pull/12404/files#diff-8f5c8742e64c355cd5201a1d735e41e4386c2bb5d0caab3236797397eaab419eR1 .

File: Packs/FeedJSON/ReleaseNotes/1_1_3.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### JSON Feed
+- **Breaking Change**: The integration now validates that not both configuration params: *Indicator Type* and *Auto detect indicator type* are set. You must set only one of the configuration params. Previously, if both were set, *Auto detect indicator type* was used and *Indicator Type* was silently ignored.
```
Comment (amshamah419) [inline Packs/FeedJSON/ReleaseNotes/1_1_3.md:4]:
```suggestion
- **Breaking Change**: The integration now validates that both configuration params: *Indicator Type* and *Auto detect indicator type* are not set. You may set only one of the configuration params at a time. Previously, if both were set, *Auto detect indicator type* was used and *Indicator Type* was silently ignored.
```

**Pull request merged**